### PR TITLE
Changes for clang-format and cpplint

### PR DESCRIPTION
**Pull request by dsuponitskiy**

### CODE REVIEW

**Comment by jbates-duality** [comment]
I've randomly sampled the changes to several files in binfhe and benchmark.

I agree with 99% of the reformatting.

I also looked at the "remove using namespace std" patch.  This appears cleanly-done.

I see that there are some conflicting files.  I have not gone down that rabbithole.  It's probably best for Dmitriy to do this.

**Comment by jbates-duality** [comment]
I have just reviewed the conflicting files.

It would appear that these are related to Ian's documentation project, and should be simple merges.

**Review by jbates-duality — APPROVED** [review]
Approving this now that conflicts are resolved and CI passed.

File: benchmark/src/compare-bfvrns-vs-bgvrns.cpp [hunk 1]
```diff
@@ -60,119 +59,119 @@ usint mult_depth = 3;
...
 
-  std::vector<int64_t> vectorOfInts = {1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
-  Plaintext plaintext = cc->MakeCoefPackedPlaintext(vectorOfInts);
+    std::vector<int64_t> vectorOfInts = {1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
+    Plaintext plaintext               = cc->MakeCoefPackedPlaintext(vectorOfInts);
 
-  vector<Ciphertext<DCRTPoly>> ciphertexts;
-  for (int i = 0; i < (1 << mult_depth); i++)
-    ciphertexts.push_back(cc->Encrypt(keyPair.publicKey, plaintext));
+    vector<Ciphertext<DCRTPoly>> ciphertexts;
```
Comment (mtriplett-duality) [inline benchmark/src/compare-bfvrns-vs-bgvrns.cpp:125]: why doesn't this require "std::"? I thought we removed the namespace up top

**Review by mtriplett-duality — APPROVED** [review]

**Comment by dsuponitskiy** [comment]
There is more cleanup for the "std::" stuff , but I will fix later. This branch initially was for reformatting only.

File: benchmark/src/compare-bfvrns-vs-bgvrns.cpp (hunk 1 above)
Comment (jbates-duality) [reply benchmark/src/compare-bfvrns-vs-bgvrns.cpp:125]: aye, yaye, yaye.  Good catch.  Regex: [^:]vector has 677 hits in the codebase.

**Pull request merged**